.travis.yml

```diff
@@ -26,6 +26,8 @@ env:
   - TESTS="1" ODOO_REPO="OCA/OCB"
   - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="website_signup_legal_page_required"
   - TESTS="1" ODOO_REPO="OCA/OCB" INCLUDE="website_signup_legal_page_required"
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="website_seo_redirection"
```

Comment:
Well, till now, we haven't done this, because it adds more time to each build, and if needed, it's done when we find a red build, but not always. You can add the lines as commented, and put a comment saying that if you suspect the module web_seo_redirection can cause the failing, uncomment that lines to try if this build is successful.